presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -385,8 +386,9 @@ protected void setup(Binder binder)
         // cost calculator
         binder.bind(CostCalculator.class).to(CostCalculatorUsingExchanges.class).in(Scopes.SINGLETON);
         binder.bind(CostCalculator.class)
-                .annotatedWith(Names.named("estimatedExchangesCostCalculator"))
+                .annotatedWith(EstimatedExchanges.class)
```

Comment:
This belongs to the previous commit. I will just change the commits and force push. No other change this time